pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -456,9 +457,9 @@ private void handleCaseSensitivity(BrokerRequest brokerRequest) {
       for (AggregationInfo info : brokerRequest.getAggregationsInfo()) {
         if (!info.getAggregationType().equalsIgnoreCase(AggregationFunctionType.COUNT.getName())) {
           // Always read from backward compatible api in AggregationFunctionUtils.
-          List<String> expressions = AggregationFunctionUtils.getAggregationExpressions(info);
+          String[] expressions = AggregationFunctionUtils.getArguments(info);
```

Comment:
Seems like AggregationFunctionUtils.getArguments first creates List, then converts to String[], and here we are doing the reverse, creating [] from List. Is that not redundant? Or is it because of more callers of getArguments() benefit if it returns String[]?

Reply:
Changed it back to List<String>. For the new format, no conversion is required. For the old backward-compatible format, there will be an array to list conversion, which is fine.